README.md

```diff
@@ -770,6 +771,23 @@ const updatedJWTRes = await descopeClient.management.jwt.update('original-jwt',
 });
 ```
 
+### Search Audit
+
+You can perform an audit search for either specific values or full-text across the fields. Audit search is limited to the last 30 days.
+
+```typescript
+// Full text search on the last 10 days
+const audits = await descopeClient.management.audit.search({
+  from: Date.now() - 10 * 24 * 60 * 60 * 1000,
```

Comment:
important example 👍 
time fields is always confusing,

moving forward, the sdk should consider supporting both unix timestamps and Date object

Reply:
Yup, even documented it in the type :-)